docs/docs/guides/labs/components/components-etl-pipeline-tutorial.md

```diff
@@ -69,20 +69,20 @@ After installing dependencies, create a components-ready Dagster project. The st
         :::
     </TabItem>
     <TabItem value="pip" label="pip">
-        Because `pip` does not support global installations, you will need to install `dg` inside your Dagster project virtual environment. To do so, follow the commands below to create and enter a Dagster project directory, initialize and activate a virtual environment, and install the `dagster-dg` package into it:
+        Because `pip` does not support global installations, you will need to install `dg` inside your Dagster project virtual environment. To do so, follow the commands below to create and enter a Dagster project directory, initialize and activate a virtual environment, and install the `dagster-dg-cli` package into it:
```

Comment:
"Install `create-dagster` using `brew` or `curl` to scaffold a new project, and then `dg` will be available in that project's virtual environment."